src/renderers/shaders/ShaderChunk/shadowmap_pars_fragment.glsl

```diff
@@ -1,12 +1,17 @@
 #ifdef USE_SHADOWMAP
 
+	uniform int isShadowCube[ MAX_SHADOWS ];
+	uniform samplerCube shadowCube[ MAX_SHADOWS ];
 	uniform sampler2D shadowMap[ MAX_SHADOWS ];
 	uniform vec2 shadowMapSize[ MAX_SHADOWS ];
 
 	uniform float shadowDarkness[ MAX_SHADOWS ];
 	uniform float shadowBias[ MAX_SHADOWS ];
 
+	uniform vec3 shadowLightPosition[ MAX_SHADOWS ];
+
 	varying vec4 vShadowCoord[ MAX_SHADOWS ];
```

Comment:
In this case, shadowLightPosition contains the world-space position of the light whereas pointLightPosition seems to contain the camera/view space position. I am currently doing all calculations for the point light shadows in world space, but I don't see why I couldn't update it to do them in camera/view space.